apps/src/studio/studio.js

```diff
@@ -3002,6 +3002,11 @@ Studio.runButtonClick = function() {
   Studio.execute();
   Studio.gameState = Studio.GameStates.ACTIVE;
 
+  if (!level.edit_blocks && level.freePlay) {
```

Comment:
Can you add a comment above this that explains when we want the special case (specifically the edit_blocks part)?